net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -17,14 +17,21 @@ start() {
 
 		exit 1
 	fi
+	config_load ampr-ripd
+	config_get enabled network enabled 0
+	if [ "$enabled" != 1 ]; then
+		echo "ampr-ripd disabled via config"
+		exit 1
+	fi
 	if [ ! -d /var/lib/ampr-ripd ]; then
 		mkdir -p /var/lib/ampr-ripd
 	fi
 	ip tunnel change ttl 64 mode ipip tunl0
 	ip link set dev tunl0 up
 	ifconfig tunl0 mtu 1480
-	tunnet=$(uci -q get ampr-ripd.network.tunnet)
-	/usr/sbin/ampr-ripd -s -r -t 44 -i tunl0 -a "$tunnet"
+	config_get amprnet network amprnet
+	config_get amprmask network amprmask
+	/usr/sbin/ampr-ripd -s -r -t 44 -i tunl0 -a "$amprnet/$amprmask"
```

Comment:
Is `-r` needed? According to the [manual](https://git.ampr.org/yo2loj/ampr-ripd/-/blob/main/manual.txt?ref_type=heads):

```
-r    Compatibility only (ignored, raw sockets are always used)
```